apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -8,48 +8,51 @@ import queryString from 'query-string';
 import color from '@cdo/apps/util/color';
 import $ from 'jquery';
 import {CommentArea} from './CommentArea';
-import TeacherFeedbackRubric from '@cdo/apps/templates/instructions/TeacherFeedbackRubric';
+import TeacherFeedbackKeepWorking from '@cdo/apps/templates/instructions/TeacherFeedbackKeepWorking';
 import TeacherFeedbackStatus from '@cdo/apps/templates/instructions/TeacherFeedbackStatus';
+import TeacherFeedbackRubric from '@cdo/apps/templates/instructions/TeacherFeedbackRubric';
 import {teacherFeedbackShape, rubricShape} from '@cdo/apps/templates/types';
+import experiments from '@cdo/apps/util/experiments';
 
 const ErrorType = {
   NoError: 'NoError',
   Load: 'Load',
   Save: 'Save'
 };
 
+const keepWorkingExperiment = 'teacher-feedback-review-state';
```

Comment:
my thinking is that it should not go there. my initial thought when i saw this was that it was one of our frontend experiments like the ones listed in `experiments.js`, which are enabled in the browser by appending `?enableExperiments=myExperiment`, but apparently this value is passed down from the server through app options.

@maureensturgeon i do think it would be helpful to add a comment here since it tripped up both James and me.